llvm/docs/CommandGuide/llvm-symbolizer.rst

```diff
@@ -370,6 +384,7 @@ OPTIONS
         "ModuleName": "inlined.elf",
         "Symbol": [
           {
+            "Approximate": false,
```

Comment:
Can we elide the "Approximate" field if the value is false?

Reply:
Done